code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1330,19 +1332,23 @@
 	ADD_TRAIT(L, TRAIT_IGNOREDAMAGESLOWDOWN, type)
 	ADD_TRAIT(L, TRAIT_NOSTAMCRIT, type)
 	ADD_TRAIT(L, TRAIT_NOLIMBDISABLE, type)
-	ADD_TRAIT(L, TRAIT_NOBLOCK, type)
+	L.visible_message("<span class='warning'>You feel like nothing can stop you!</span>")
 
 /datum/reagent/stimulum/on_mob_end_metabolize(mob/living/L)
 	REMOVE_TRAIT(L, TRAIT_STUNIMMUNE, type)
 	REMOVE_TRAIT(L, TRAIT_SLEEPIMMUNE, type)
 	REMOVE_TRAIT(L, TRAIT_IGNOREDAMAGESLOWDOWN, type)
 	REMOVE_TRAIT(L, TRAIT_NOSTAMCRIT, type)
 	REMOVE_TRAIT(L, TRAIT_NOLIMBDISABLE, type)
-	REMOVE_TRAIT(L, TRAIT_NOBLOCK, type)
+	L.visible_message("<span class='warning'>You can feel your brief high wearing off</span>")
 	..()
 
 /datum/reagent/stimulum/on_mob_life(mob/living/carbon/M)
 	M.adjustStaminaLoss(-2*REM, 0)
+	M.losebreath += (current_cycle*0.05) // gradually builds up suffocation, will not be noticable for several ticks but effects will linger afterwards
```

Comment:
This should definitely be clamped, being able to stack this would be messy

Reply:
Put two conditions on it:
 * Capped at +2 losebreath per cycle
 * Will not trigger if you have more than 10
 
 The cap per cycle ensures Stimulum will never outpace Epinephrine's ability to counteract it, and putting a lid on how much is possible ensures nobody ends up with 60+ seconds of suffocation stacked up.

Reply:
isn't losebreath a timer rather than an amount lost per second, so you will always be able to outpace it?